config/configChecker_test.go

```diff
@@ -276,6 +276,32 @@ func TestSanityCheckNodesConfig(t *testing.T) {
 		}
 		err = SanityCheckNodesConfig(nodesSetup, cfg)
 		require.Nil(t, err)
+
+		cfg.MaxNodesChangeEnableEpoch = []MaxNodesChangeConfig{
```

Comment:
can we add this configuration also? It is the testnet config:
```
  MaxNodesChangeEnableEpoch = [
        { EpochEnable = 0, MaxNumNodes = 2169, NodesToShufflePerShard = 143 },
        { EpochEnable = 1, MaxNumNodes = 3200, NodesToShufflePerShard = 80 }
    ]
```

Reply:
Yes, it needs an extra entry for EnableEpoch = StakingV4Step3 (which is 6 in this unit test) to pass
Added unit test